client/src/app/submission/submission.component.html

```diff
@@ -28,6 +28,39 @@ <h2 class="vf-text vf-text-heading--2">
   <div>
     <div class="vf-text-body vf-text-body--2">{{projectTitle}}</div>
     <div class="vf-text-body vf-text-body--3">{{getContributors(project)}}</div>
+    <div *ngIf="validationSummary">
+      <p>Your metadata validation returned <span class="error-text">{{validationSummary.totalInvalid}} error(s)</span>. Review and fix them below.</p>
+
+      <ul id="validation-error-types">
+        <li>
+          Biomaterials:
+          <a href="javascript: void(0)" (click)="navigateToTab(0, { dataSource: biomaterialsDataSource, filterState: 'Invalid'})">
+            {{validationSummary.invalidBiomaterials}} error(s)
+          </a>
+        </li>
+        <li>
+          Processes:
+          <a href="javascript: void(0)"(click)="navigateToTab(1, { dataSource: processesDataSource, filterState: 'Invalid' })">
+            {{validationSummary.invalidProcesses}} error(s)
+          </a>
+        </li>
+        <li>
+          Protocols:
+          <a href="javascript: void(0)"(click)="navigateToTab(2, { dataSource: protocolsDataSource, filterState: 'Invalid' })">
+            {{validationSummary.invalidProtocols}} error(s)
+          </a>
+        </li>
+        <li>
+          Data files:
+          <a href="javascript: void(0)" (click)="navigateToTab(3, { dataSource: filesDataSource, filterState: 'Invalid'})">
```

Comment:
I think this should be 'Invalid File'

Reply:
This raises another issue. In fact, I think we would need "Invalid File", "Invalid Metadata", and "Missing File" right? The summary endpoint only returns counts for invalidFiles and missingFiles

Reply:
@prabh-t is adding these properties to the summary endpoint. I will check on this with @gabsie later on and implement per her specs